### PR TITLE
Allow to load data from IEX

### PR DESCRIPTION
**Pull request by DidierRLopes**
I went over the credits of IEX. I don't think it has a great free tier.

But anyway, it has a lot of stuff that we can perhaps integrate at some point. So probably worth having this as a starting point. See https://github.com/iexcloud/pyEX/blob/main/examples/all.ipynb

This PR includes:
  * IEX daily and intradaily data
  * Fix view help
  * Fix load help
  * View function to allow to print raw data
  * Remove export function and refactor it into a flag in view function
  * Add exports/ folder to gitignore
  * Improve FTD command by allowing to select a start/end date + plotting share price

### CODE REVIEW

File: gamestonk_terminal/main_helper.py [hunk 1]
```diff
@@ -194,6 +195,38 @@ def load(other_args: List[str], s_ticker, s_start, s_interval, df_stock):
...
+                        "volume": "Volume",
+                    }
+                )
+
+                df_stock_candidate.sort_index(ascending=True, inplace=True)
+
+                # Slice dataframe from the starting date YYYY-MM-DD selected
+                df_stock_candidate = df_stock_candidate[ns_parser.s_start_date :]
+
+                print(df_stock_candidate.to_string())
```
Comment (jmaslek) [inline gamestonk_terminal/main_helper.py:228]: Do you want this to print?

**Review by jmaslek — APPROVED** [review]

**Review by jmaslek — COMMENTED** [review]
Also intraday doesn't work  (note `load msft --source iex` works)

Screen Shot 2021-07-09 at 10 28 59 AM

**Review by jmaslek — CHANGES_REQUESTED** [review]
So the intraday option is not available for iex as written.  Not sure if you want to have a flag that says that or add it in (I think the `client.chart` has some extra kwargs for intraday (https://github.com/iexcloud/pyEX/blob/d9c38826ebc5c35ffb80903f006834a5989d70c5/pyEX/stocks/prices.py#L87))

File: gamestonk_terminal/main_helper.py (hunk 1 above)
Comment (DidierRLopes) [reply gamestonk_terminal/main_helper.py:228]: Nahhh

**Comment by DidierRLopes** [comment]
> So the intraday option is not available for iex as written. Not sure if you want to have a flag that says that or add it in (I think the `client.chart` has some extra kwargs for intraday (https://github.com/iexcloud/pyEX/blob/d9c38826ebc5c35ffb80903f006834a5989d70c5/pyEX/stocks/prices.py#L87))

Yea, the intraday only gives 1m for the day... I'm gonna try to play with that

**Comment by DidierRLopes** [comment]
@jmaslek I've added a couple of things to this PR :) Should be in a good shape now

File: gamestonk_terminal/main_helper.py (hunk 1 above)
Comment (jmaslek) [reply gamestonk_terminal/main_helper.py:228]: Still does

**Comment by jmaslek** [comment]
After I do `load AAPL -i 1 --source iex` , the `view` is weird

**Review by jmaslek — APPROVED** [review]

**Review by aia — APPROVED** [review]

**Pull request merged**